CRM/Contribute/BAO/Contribution.php

```diff
@@ -4043,19 +4045,13 @@ public static function getPaymentInfo($id, $component = 'contribution', $getTrxn
       $contributionId = $id;
     }
 
-    $total = CRM_Core_BAO_FinancialTrxn::getBalanceTrxnAmt($contributionId);
-    $baseTrxnId = !empty($total['trxn_id']) ? $total['trxn_id'] : NULL;
+    $oldCalculation = CRM_Core_BAO_FinancialTrxn::getBalanceTrxnAmt($contributionId);
```

Comment:
@eileenmcnaughton It would be helpful to have a comment explaining why this variable is called `oldCalculation` as it's a mystery to me. Non-blocking to merging of this PR but if you get chance.

Reply:
Oh - because I think it is weird & should go

Reply:
@mattwire  - I've expanded the code comments